styles.css

```diff
@@ -0,0 +1,71 @@
+article {
+	border: 1px solid lightblue;
+	width: 30%;
+	display: inline-block;
+	margin-bottom: 20px;
+	font-family: Arial, Helvetica, sans-serif;
+}
+
+img{
+	padding-left: 3.25em;
```

Comment:
putting padding on an image to center works...until you change the screen size then the image spills over the screen and looks a little funky.  Can you think of another way to do this?  hint: set the image size to a percentage and change the alignment on the parent div of the image